packages/babel-plugin-transform-dotall-regex/test/index.js

```diff
@@ -0,0 +1,24 @@
+import path from 'path';
+import fs from 'fs';
+import assert from 'assert';
+import { transform, transformFileSync } from '@babel/core';
```

Comment:
`transform` is not used, we can remove it.

Reply:
Done.